packages/components/src/components/link-social/link-social.tsx

```diff
@@ -51,11 +53,24 @@ export class LinkSocial {
 
   public componentWillLoad(): void {
     throwIfInvalidLinkUsage(this.host, this.href);
+    throwIfComponentIsDeprecated(
```

Comment:
A generic util would take the new component name as the 2nd parameter and the message would be part of the util.

Reply:
Yeah, but there is not always a "new" component. SOmetimes a component is just "deprecated" and nothing else. So we need a possibility of customization in the warning message.